arcsi/api/item.py

```diff
@@ -341,19 +349,19 @@ def download_play_file(id):
     return redirect(presigned, code=302)
 
 
-@arcsi.route("/item/<int:id>", methods=["DELETE"])
+@arcsi.route("/archon/item/<int:id>", methods=["DELETE"])
 @roles_required("admin")
-def delete_item(id):
+def archon_delete_item(id):
```

Comment:
Is it working now?

Reply:
Nope, I didn't add this yet, maybe in the next PR.

Reply:
Also, it could be mentioned in an in-code comment that it's not working yet, just a placeholder.

Reply:
True, I added a TODO for the DELETE APIs.